modules/sharedtest/testobjects/vm.go

```diff
@@ -39,7 +40,7 @@ func newRandomVMIWithNS(namespace string) *v1.VirtualMachineInstance {
 	return vmi
 }
 
-func newRandomVirtualMachine(vmi *v1.VirtualMachineInstance, running bool) *v1.VirtualMachine {
+func newRandomVirtualMachine(vmi *v1.VirtualMachineInstance, runStrategy *v1.VirtualMachineRunStrategy) *v1.VirtualMachine {
```

Comment:
Looking at it again: Like this you should be able to get away without adding any dependency.

```suggestion
func newRandomVirtualMachine(vmi *v1.VirtualMachineInstance, runStrategy v1.VirtualMachineRunStrategy) *v1.VirtualMachine {
```